R/antigen.garnish_predict.R

```diff
@@ -1655,9 +1655,14 @@ up <- lapply(dtl, function(x){x[[2]]}) %>% unlist
      if ("blast_uuid" %chin% names(dt))
       dt[!is.na(BLAST_A), min_DAI := BLAST_A]
 
-    if ("DAI" %chin% names(dt)) dt[!is.na(DAI),
-                                  min_DAI := min(DAI, min_DAI, na.rm = TRUE),
-                                  by = 1:nrow(dt[!is.na(DAI)])]
+    if ("DAI" %chin% names(dt)){
```

Comment:
why did this vector need to be explicitly extracted vs. by =  1:nrow(dt[!is.na(DAI)])?

perhaps we should set a reasonable upper and lower DAI bound vs. calculating like this since these extrema can vary pretty dramatically across n=low experiments? it is normally distributed (cf TCGA)

Reply:
Because it allowed me to control when that value is zero, because in that edge case the vector becomes 1:0, which is length 2, and the table is nrow = 0 so it throws an error because the by call is longer than nrow dt. I could have just put an if statement in before that line, but originally I thought it was an issue with the column name being mistaken for a character based on the error it returned. up to you if you want to change. I think slightly more readable this way maybe.